app/scripts/modules/core/src/pipeline/config/pipelineConfig.controller.spec.js

```diff
@@ -29,6 +29,11 @@ describe('Controller: PipelineConfigCtrl', function() {
       },
       app: application,
     });
+    application.pipelineConfigs.activate();
```

Comment:
The description of this test case is `'should initialize immediately if pipeline configs are already present'`. This change is the opposite of this - it's saying that "even if the pipeline configs are already loaded, I want to reload them". Would delete this test case and include a different one that demonstrates that behavior.